MovieMVVM/MovieMVVMTests/AssemblyModuleBuilderTests.swift

```diff
@@ -0,0 +1,54 @@
+// AssemblyModuleBuilderTests.swift
+// Copyright © KarpovaAV. All rights reserved.
+
+@testable import MovieMVVM
+import XCTest
+
+/// Тест сборщика модулей
+final class AssemblyModuleBuilderTests: XCTestCase {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let idNumber = 315_162
+    }
+
+    // MARK: - Private Properties
+
+    private var assemblyModuleBuilder: AssemblyBuilderProtocol?
+
+    // MARK: - Public Methods
+
+    override func setUp() {
```

Comment:
```suggestion
    override func setUp() {
```
Надо super добавлять